src/ape_http/providers.py

```diff
@@ -35,24 +129,64 @@ class NetworkConfig(ConfigItem):
     ethereum: EthereumNetworkConfig = EthereumNetworkConfig()
 
 
+class GethNotInstalledError(ConnectionError):
+    def __init__(self):
+        super().__init__(
+            "geth is not installed and there is no local provider running.\n"
+            "Things you can do:\n"
+            "\t1. Install geth and try again\n"
+            "\t2. Run geth separately and try again\n"
+            "\t3. Use a different ape provider plugin"
+        )
+
+
 class EthereumProvider(ProviderAPI):
```

Comment:
Might make sense to rename to `GethProvider`

Reply:
Should we also rename the plugin to `ape-geth` ?